src/VisualStudio/IntegrationTest/TestUtilities/ScreenshotService.cs

```diff
@@ -28,12 +33,23 @@ public static void TakeScreenshot(string fullPath)
         /// <summary>
         /// Captures the full screen to a <see cref="Bitmap"/>.
         /// </summary>
-        /// <returns>A <see cref="Bitmap"/> containing the screen capture of the desktop.</returns>
-        private static Bitmap CaptureFullScreen()
+        /// <returns>
+        /// A <see cref="Bitmap"/> containing the screen capture of the desktop, or null if a screen
+        /// capture can't be created.
+        /// </returns>
+        private static Bitmap TryCaptureFullScreen()
         {
-            var bitmap = new Bitmap(
-                width: Screen.PrimaryScreen.Bounds.Width,
-                height: Screen.PrimaryScreen.Bounds.Height);
+            int width = Screen.PrimaryScreen.Bounds.Width;
+            int height = Screen.PrimaryScreen.Bounds.Height;
+
+            if (width <= 0 || height <= 0)
+            {
+                // Don't try to take a screenshot if there is no screen.
```

Comment:
So something has already gone wrong, and as a side effect of that we're trying to take a screenshot. I don't really want to fail the test while taking the screenshot as that is a distraction from the real problem.